pr-data.csv

```diff
@@ -947,8 +947,8 @@ https://github.com/FasterXML/jackson-dataformats-binary,d78d13c574a4b139438841bd
 https://github.com/FasterXML/jackson-dataformats-binary,d78d13c574a4b139438841bd5859b8b99beb63ea,avro,com.fasterxml.jackson.dataformat.avro.interop.annotations.AvroEncodeTest.testUuidValue[Apache to Jackson with Jackson schema],ID,,,
 https://github.com/FasterXML/jackson-dataformats-binary,d78d13c574a4b139438841bd5859b8b99beb63ea,avro,com.fasterxml.jackson.dataformat.avro.interop.annotations.AvroEncodeTest.testUuidValue[Jackson to Jackson with Apache schema],ID,,,
 https://github.com/FasterXML/jackson-dataformats-binary,d78d13c574a4b139438841bd5859b8b99beb63ea,avro,com.fasterxml.jackson.dataformat.avro.interop.annotations.AvroEncodeTest.testUuidValue[Jackson to Jackson with Jackson schema],ID,,,
-https://github.com/FasterXML/jackson-datatypes-collections,8b88868ac3012783fb7f1bda00309c823172e02a,guava,com.fasterxml.jackson.datatype.guava.FluentIterableTest.testSerialization,ID,,,
-https://github.com/FasterXML/jackson-datatypes-collections,8b88868ac3012783fb7f1bda00309c823172e02a,guava,com.fasterxml.jackson.datatype.guava.FluentIterableTest.testWrappedSerialization,ID,,,
+https://github.com/FasterXML/jackson-datatypes-collections,8b88868ac3012783fb7f1bda00309c823172e02a,guava,com.fasterxml.jackson.datatype.guava.FluentIterableTest.testSerialization,ID,InspiredAFix,https://github.com/FasterXML/jackson-datatypes-collections/pull/91,https://github.com/TestingResearchIllinois/idoft/issues/48
+https://github.com/FasterXML/jackson-datatypes-collections,8b88868ac3012783fb7f1bda00309c823172e02a,guava,com.fasterxml.jackson.datatype.guava.FluentIterableTest.testWrappedSerialization,ID,InspiredAFix,https://github.com/FasterXML/jackson-datatypes-collections/pull/91,https://github.com/TestingResearchIllinois/idoft/issues/48
```

Comment:
These two columns are missing a comma.
```InspiredAFix,https:``` -> ```InspiredAFix,,https:```

Reply:
Please ignore the preceding comment. The problem with the failed build appears to be an issue with the checker.